feg/gateway/services/session_proxy/session_proxy/main.go

```diff
@@ -68,6 +68,9 @@ func main() {
 	// Add servicers to the service
 	sessionManagerAndHealthServer, err := servicers.
 		NewCentralSessionControllerDefaultMultiplexWithHealth(controllerParms, policyDBClient)
+	if err != nil {
+		glog.Fatalf("Could not add Health Server to servicer: %s", err)
+	}
```

Comment:
Does it have to be 'Fatalf' here?
Will this work?

```
	if err != nil {
		glog.Errorf("Could not add Health Server to servicer: %s", err)
	} else {
	        lteprotos.RegisterCentralSessionControllerServer(srv.GrpcServer, sessionManagerAndHealthServer)
	        protos.RegisterServiceHealthServer(srv.GrpcServer, sessionManagerAndHealthServer)
        }	
```

In a single FeG instance, even an 'unhealthy' FeG should mostly function?

Reply:
Note that `NewCentralSessionControllerDefaultMultiplexWithHealth` returns the single instance or multiple (depending on the config).

So if there is an error,  sessionManagerAndHealthServer will be nil, so no grpc server will be able to be run. The service will be stuclk. 

At least if we crashloop we will be able to see it. And now with this new feature 
 at https://github.com/magma/magma/pull/6288 we may be able to catch those crashes before even landing